src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -27,5 +28,17 @@ public sealed class NotificationHubAttribute : Attribute
         /// A tag expression is any boolean expression constructed using the logical operator
```

Comment:
Can you add Optional in comments for TagExpression as well. If not set, it sends broadcast